# HG changeset patch
# User Rik <rik@octave.org>
# Date 1382646891 25200
#      Thu Oct 24 13:34:51 2013 -0700
# Node ID 031eaa90064e96385c747dc96c661dfbe576f91b
# Parent  e2f83152e64a2c1147a26da9590d7422d90b04a0
NEWS: Update announcements for 3.8 release.

* NEWS: Update announcements for 3.8 release.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,27 +1,16 @@
 <pre>
-Summary of important user-visible changes for version 4.0:
+Summary of important user-visible changes for version 3.8:
 ---------------------------------------------------------
 
- ** A new GUI is now available with Octave.
-
- ** Warning states may now be set temporarily, until the end of the
-    current function, using the syntax
-
-      warning STATE ID "local"
-
-    in which STATE may be "on", "off", or "error".  Changes to warning
-    states that are set locally affect the current function and all
-    functions called from the current scope.  The previous warning state
-    is restored on return from the current function.  The "local"
-    option is ignored if used in the top-level workspace.
-
-Summary of important user-visible changes for version 3.8:
----------------------------------------------------------
+ ** A GUI is now available with Octave and is the default when starting
+    an interactive session.  The original command line interface is also
+    available and can be entered by using the "--no-gui" option when
+    invoking Octave.
 
  ** Octave now uses OpenGL graphics by default with FLTK widgets.  If
     OpenGL libraries or FLTK widgets are not available when Octave is
     built, gnuplot is used.  You may also choose to use gnuplot for
     graphics by executing the command
 
       graphics_toolkit ("gnuplot")
 
@@ -105,16 +94,27 @@ Summary of important user-visible change
       catch myerr
         printf ("identifier: %s\n", myerr.identifier);
         printf ("message:    %s\n", myerr.message);
       end_try_catch
 
     When classdef-style classes are added to Octave, the exception-id
     will become an MException object.
 
+ ** Warning states may now be set temporarily, until the end of the
+    current function, using the syntax
+
+      warning STATE ID "local"
+
+    in which STATE may be "on", "off", or "error".  Changes to warning
+    states that are set locally affect the current function and all
+    functions called from the current scope.  The previous warning state
+    is restored on return from the current function.  The "local"
+    option is ignored if used in the top-level workspace.
+
  ** Warning IDs renamed:
 
     Octave:array-as-scalar => Octave:array-to-scalar
     Octave:array-as-vector => Octave:array-to-vector
 
  ** 'emptymatch', 'noemptymatch' options added to regular expressions.
 
     With this addition Octave now accepts the entire set of Matlab options
@@ -153,17 +153,17 @@ Summary of important user-visible change
     \fontname    font        \fontname{Arial}    set Arial font
     \fontsize    fontsize    \fontsize{16}       set fontsize 16
     \color[rgb]  fontcolor   \color[rgb]{1 0 1}  set magenta color 
     \bf          bold        \bfBold Text        bold font
     \it          italic      \itItalic Text      italic font
     \sl          slanted     \slOblique Text     slanted font
     \rm          normal      \bfBold\rmNormal    normal font
     {}           group       {\bf Bold}Normal    group objects
-                             e^{i*\pi} = -1      complex example
+                             e^{i*\pi} = -1      group objects
 
  ** The m-files in the plot directory have been overhauled.
 
     The plot functions now produce output that is nearly visually compatible
     with Matlab.  Plot performance has also increased, dramatically for some
     functions such as comet and waitbar.  Finally, the documentation for most
     functions has been updated so it should be clearer both how to use a
     function and when a function is appropriate.
@@ -300,22 +300,22 @@ Summary of important user-visible change
       betaincinv                 flintmax         polyeig
       built_in_docstrings_file   fminsearch       prefdir
       cmpermute                  gallery          preferences
       cmunique                   gco              readline_re_read_init_file
       colorcube                  hdl2struct       readline_read_init_file
       copyobj                    history_save     rgbplot
       dawson                     imformats        save_default_options
       dblist                     importdata       shrinkfaces
-      debug_jit                  isaxes           splinefit
-      desktop                    iscolormap       stemleaf
-      doc_cache_create           isequaln         strjoin
-      ellipj                     jit_debug        struct2hdl
-      ellipke                    jit_enable       tetramesh
-      erfcinv                    jit_startcnt     waterfall
+      desktop                    isaxes           splinefit
+      doc_cache_create           iscolormap       stemleaf
+      ellipj                     isequaln         strjoin
+      ellipke                    jit_debug        struct2hdl
+      erfcinv                    jit_enable       tetramesh
+                                 jit_startcnt     waterfall
 
  ** Deprecated functions.
 
     The following functions were deprecated in Octave 3.4 and have been
     removed from Octave 3.8.
                                            
       autocor    dispatch              is_global    setstr
       autocov    fstat                 krylovb      strerror
